theme/assets/js/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+window.addEventListener('load', function () {
+  var versionSelect = document.getElementById('version-select');
+  var regexp = /version\/.+\//;
+  versionSelect.value = location.pathname
+    .match(regexp)[0]
+    .replace('version', '')
+    .replace(/\//g, '');
+
+  versionSelect.addEventListener('change', function () {
+    window.location.replace(location.href.replace(regexp, 'version/' + this.value));
```

Comment:
IMO, it could make more sense to associate `option`s with links during rendering (i.e., in the `header` partial) in order to better encapsulate front-end code. For example, this could be done via an attribute (e.g., `data-href`).

Reply:
@slowli it will be hard code for specific paths, this solution is more flexible for localhost / another domains